hedera-mirror-rest-java/src/test/java/com/hedera/mirror/restjava/controller/AllowancesControllerIntegrationTest.java

```diff
@@ -72,7 +74,34 @@ void setup() {
 
     @Test
     void successWithNoQueryParamsShardRealmNumAccountId() {
-        var allowance1 = domainBuilder.nftAllowance().persist();
+        var entity = domainBuilder.entity().persist();
+        var allowance1 = domainBuilder
+                .nftAllowance()
+                .customize(nfta -> nfta.owner(entity.getId()))
+                .persist();
+        var allowance2 = domainBuilder
+                .nftAllowance()
+                .customize(nfta -> nfta.owner(allowance1.getOwner()))
+                .persist();
+        Collection<NftAllowance> collection = List.of(allowance1, allowance2);
+
+        var result = restClient
+                .get()
+                .uri("", allowance1.getOwner())
+                .accept(MediaType.ALL)
+                .retrieve()
+                .body(NftAllowancesResponse.class);
+        assertThat(result.getAllowances()).isEqualTo(mapper.map(collection));
+        assertNull(result.getLinks().getNext());
```

Comment:
Always use assertj. Check elsewhere.

```suggestion
        assertThat(result.getLinks().getNext()).isNull();
```